aws-ts-appsync/index.ts

```diff
@@ -43,8 +44,13 @@ const apiKey = new aws.appsync.ApiKey("key", {
     apiId: api.id,
 });
 
+const randomString = new random.RandomString("random-datasource-name", {
+    length: 15,
```

Comment:
You should limit this to `[_A-Za-z][_0-9A-Za-z]`, i.e. exclude special characters from `RandomString` options.

Reply:
👍 good call!